tests/test_register_existing.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+import securemailbox
+
+test_fingerprint = "FAC10F0C3D1D49F8F9A82CB553E79F7C92E1CF33"
+
+#incoming_fingerprint = request.json.get("fingerprint")
+#new_mailbox = Mailbox(fingerprint=test_fingerprint)
+#db.session.add(new_mailbox)
+#db.session.commit()
```

Comment:
I think you could move this into the test function and and eliminate the first register on line 14. Might be the better way to do it even though I think it was me who initially wrote it with the two registers.

Reply:
It works now? I don't think it needs to be interfered with...

Reply:
Understandable. Can you remove the commented out code then?